content/en/docs/Writing policies/match-exclude.md

```diff
@@ -13,7 +13,7 @@ The `match` and `exclude` clauses have the same structure and can each contain t
 * `roles`: select namespaced roles
 * `clusterRoles`: select cluster wide roles
 
-At least one element must be specified in a `match` or `exclude` block. The `kind` attribute is optional when working with the `resources` element, but if it's not specified the policy rule will only be applicable to metadata that is common across all resource kinds.
+At least one element must be specified in a `match.resources.kinds` or `exclude` block. The `kind` attribute is mandatory when working with the `resources` element. Currently not supporting  wildcard `*` for `match.resources.kinds`
```

Comment:
"Wildcards (`*`) are currently not supported in the `match.resources.kinds` field."